packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx

```diff
@@ -106,7 +109,11 @@ export function TimeRangePicker(props: TimeRangePickerProps) {
       <Tooltip content={<TimePickerTooltip timeRange={value} timeZone={timeZone} />} placement="bottom" interactive>
         <ToolbarButton
           data-testid={selectors.components.TimePicker.openButton}
-          aria-label={`Time range picker with current time range ${formattedRange(value, timeZone)} selected`}
+          aria-label={t(
+            'time-range.range-picker.current-time-selected',
+            'Time range picker with current time range {{currentTimeRange}} selected',
```

Comment:
i think this might be a good time to make this label more concise. Lets see what happens from this thread https://raintank-corp.slack.com/archives/C01JZE87FQ9/p1668169990393939